packages/fxa-auth-server/test/local/routes/subscriptions.js

```diff
@@ -2319,7 +2319,92 @@ describe('DirectStripeRoutes', () => {
     });
   });
 
-  describe('updatePayment', () => {});
+  describe('updatePayment', () => {
+    let request;
+    const paymentToken = 'tok_visa';
+
+    beforeEach(() => {
+      request = Object.create(VALID_REQUEST);
+      request.payload = { paymentToken };
```

Comment:
Any reason you favor `Object.create` over something like `request = {...VALID_REQUEST, payload: {paymentToken}};`?

Reply:
No reason in particular, it was just where my mind went. Updating.